src/_guides/language/analysis-options.md

```diff
@@ -70,15 +70,15 @@ to perform static analysis.
 You can customize static analysis to look for a variety of potential
 problems, including errors and warnings specified in the
 [Dart language spec](/guides/language/spec).
-You can also configure the linter, one of the analyzer's plugins,
+You can also configure linter rules,
 to ensure that your code complies with the
 [Dart Style Guide](/guides/language/effective-dart/style)
 and other suggested guidelines in
 [Effective Dart][]. Dart tools such as the
 [Dart compiler (`dart compile`)](/tools/dart-compile),
 [`dart analyze`](/tools/dart-analyze),
 [`flutter analyze`]({{site.flutter-docs}}/testing/debugging#the-dart-analyzer),
-and [JetBrains IDEs](/tools/jetbrains-plugin)
+and [Editors and IDEs](/tools#ides-and-editors)
```

Comment:
For some reason this order sounds slightly better to me, and matches the linked section.

```suggestion
and [IDEs and editors](/tools#ides-and-editors)
```